apps/docs/js-interpreter.md

```diff
@@ -0,0 +1,167 @@
+# Upgrading the JS Interpreter
+
+## Component Overview
+
+Before embarking on an upgrade of the JS interpreter,
+you need to know what all the different little pieces
+are and how they relate to each other.
+
+### Neil Fraser's JS-Interpreter Repository
+
+[NeilFraser/JS-Intepreter](https://github.com/NeilFraser/JS-Interpreter)
+is the main repository where development of the interpreter takes place. It gets updated
+sporadically every 3-6 months or so. The goal of "upgrading the interpreter" is about
+getting the updates in this repository to work on code.org.
+
+### Code.org's "fork" of Neil's JS-Interpreter
```

Comment:
hmm, I guess that's true. I was thinking of fork in the classical sense, not in the "fork me on github" sense. Will remove the quotes.